xcp_d/workflows/anatomical/volume.py

```diff
@@ -168,7 +167,6 @@ def init_postprocess_anat_wf(
                 ),
                 name="warp_t1w_to_template",
                 mem_gb=2,
-                n_procs=omp_nthreads,
```

Comment:
I would keep this and set ``num_threads=omp_nthreads`` in the ApplyTransforms in the interface